### ISSUE TITLE
[BITV] 9.1.4.3/1.2 - When using a user stylesheet (html {background-color:black;color:white}), the sentence "Nextcloud - a secure place for all your data" at the bottom of the page (white text) is no longer properly readable/discernible. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36950]
on login page 

- [ ] Fix
- [ ] Backport

![image](https://user-images.githubusercontent.com/24800714/222469565-f5e82350-f27b-43f7-98af-e4b74f29520b.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v1-n2

</details>

**Comment by susnux** [comment]
@nimishavijay Not sure if the pill style footer, like on your mockup ( https://github.com/nextcloud/server/issues/35482#issuecomment-1534723358 ), works out when configuring GDPR or legal information links. What do you think?

rounded | box small | box wide
---|---|---
![pill style with more information](https://user-images.githubusercontent.com/1855448/236221049-bec75172-b9e6-4966-b088-bab567a95726.png)|![Footer aligned with login form](https://user-images.githubusercontent.com/1855448/236218887-df55665d-b9d5-4363-b20f-c331606e669e.png) |![full width footer](https://user-images.githubusercontent.com/1855448/236218926-6f9bd351-e159-481d-9a5c-3ca792fe4ebf.png)

(I could bet I somewhere already posted this mockups...)

**Comment by susnux** [comment]
One could even unify the look with the dashboard which would look like this:

![using translucent blurry background](https://user-images.githubusercontent.com/1855448/236223621-ec3937ad-eb3a-436b-be3f-6854eb5764cd.png)

**Comment by jancborchardt** [comment]
@susnux looks nice – 2 adjustments:
- [ ] The radius of the main container then also needs to be increased to the one in the Dashboard
- [ ] The Dashboard doesn’t only have transparency but also CSS blur, that needs to be there as well for proper contrast.

**Comment by susnux** [comment]
@jancborchardt yes I forgot to add the `backdrop-filter` property on that mockup (updated it).

**Comment by nimishavijay** [comment]
Yes, nice idea to align it with the dashboard! :) 
I would add that the pill style container on the bottom can have more padding on the left and right -- around 4-8px more.
